app/views/content/funding-and-support/salaried-teacher-training.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Salaried teacher training"
+description: |-
+    Explore salaried teacher training, including School Direct (salaried) courses, postgraduate teaching apprenticeships and training delivered by Teach First.
+related_content:
+    Nathan's experience of salaried teacher training: "/blog/salaried-teacher-training-classroom-learning"
+    If you've worked as an unqualified teacher: "/train-to-be-a-teacher/assessment-only-route-to-qts"
+    What to expect on your teacher training: "/blog/what-to-expect-on-your-teacher-training"
+promo_content:
+    - content/train-to-be-a-teacher/promos/find-your-course
+    - content/funding-and-support/promos/get-adviser-funding-promo
+navigation: 20.20
+navigation_description: Find out about salaried teacher training, including School Direct (salaried) courses, postgraduate teaching apprenticeships and training delivered by Teach First.
+keywords:
+    - Teach First
+    - School Direct
+    - School Direct (salaried)
+    - Postgraduate teaching apprenticeship
+    - PGTA
+    - Salaried courses
+    - Salaried teacher training
+    - Salaried teacher training course
+    - Paid teacher training
+---
+
+There are some teacher training courses that come with a salary. 
+
+This means you will not have to pay tuition fees and will receive a salary while you train to get [qualified teacher status (QTS)](/what-is-qts). This is what you need to teach in many primary, secondary and special schools in England. 
+
+The main salaried courses include School Direct (salaried), postgraduate teaching apprenticeships (PGTA) and teacher training delivered by Teach First.
+
+All salaried courses are broadly the same, but the course length and qualifications you work towards may vary.
+
+They're in high demand and very competitive, so it’s important to apply as soon as you can if you’re eligible. 
+
+## Am I eligible for a salaried teacher training course? 
+
+For some salaried courses, teacher training providers may want you to have significant teaching or school experience (for example, if you’re already working as an unqualified teacher or teaching assistant).  
+
+Some also want you to have a school lined up to work in while you train. Speak to the teacher training provider to find out what their requirements are. 
+
+You can also [find out more about the qualifications you need to apply for all teacher training courses](/is-teaching-right-for-me/qualifications-you-need-to-teach). 
+
+Or talk to a [teacher training adviser](/teacher-training-advisers) who can help you understand which courses you could be eligible for. 
+
+## How much will I be paid on a salaried course? 
+
+You'll receive an [unqualified teacher’s salary](/salaries-and-benefits#unqualified-teacher-salaries) – the exact amount will vary depending on the school you work in. 
```

Comment:
Normally we avoid links to anchors - however, I think this is probably appropriate here?

Reply:
Just thought - the important information re pay award, which year this applies to etc. is at the top of the Salaries page. By linking to an anchor, we are not giving people a chance to see this information @EllieNodder ?

Reply:
good point - shall remove the anchor and just leave overall link in